docs/docs/community/faq.md

```diff
@@ -10,3 +10,29 @@ Novu consists of multiple services written in Node.js and Typescript. The follow
 - MongoDB
 - Redis
 - File storage (S3/GCP/Azure) - Optional
+
+## How do I customize messages on Novu?
+
+In Novu, [handlebar variables](https://handlebarsjs.com/guide/) (variables enclosed within double curly brackets) are used to customize messages. Using this, one can take advantage of the following tools:
+
+- **Custom Variables:** You can create [custom variables](https://docs.novu.co/platform/templates#variable-usage) by defining it in the payload and access the value of the same using the following syntax:
```

Comment:
```suggestion
- **Custom Variables:** You can create [custom variables](https://docs.novu.co/platform/templates#variable-usage) by defining it in the payload and accessing its value using the following syntax:
```

What do you think? Something feels off to me in the sentence 'the value of the same'🤔

Reply:
Yeah, this sounds a lot more natural! ✅